README.md

```diff
@@ -33,13 +33,26 @@ It's listed on [Elixir - Tools and Data Services Registry](https://bio.tools/nf-
 
 By default, the pipeline currently performs the following:
 
-- Sequencing quality control (`FastQC`)
-- Map Reads to Reference (`BWA mem`)
-- Mark Duplicates (`GATK MarkDuplicates`)
-- Base (Quality Score) Recalibration (`GATK BaseRecalibrator`, `GATK ApplyBQSR`)
-- Preprocessing quality control (`samtools stats`)
-- Preprocessing quality control (`mosdepth`)
-- Overall pipeline run summaries (`MultiQC`)
+- Form consensus reads from UMI sequences (`fgbio`)
```

Comment:
this is not donw by default

Reply:
Don't really have an opinion :D Just because at the top it says: run by default. I think the phrasing should fit together